Core/Install/elxBaseComponent.cxx

```diff
@@ -88,7 +90,7 @@ void BaseComponent::InitializeElastixExecutable()
 
 std::string
 BaseComponent::ConvertSecondsToDHMS(
-  const double totalSeconds, const unsigned int precision = 0 ) const
+  const double totalSeconds, const unsigned int precision )
```

Comment:
are you sure removing the default here does not give issues?

Reply:
Thanks for asking, Marius. Yes, I'm sure it's OK to remove the default (`=0`) here, because it is in the CXX file, so it does not affect the interface! (I think the default was a mistake here. For the default to have any effect, it should have been in the H file.)